packages/database/src/entities/JudgingSession.ts

```diff
@@ -0,0 +1,31 @@
+import { ManyToOne, Property, Entity, Ref, EntityDTO } from '@mikro-orm/core';
+import { v4 } from 'uuid';
+import { Node } from './Node';
+import { ConstructorValues } from '../types/ConstructorValues';
+import { User } from './User';
+
+export type JudgingSessionDTO = EntityDTO<JudgingSession>;
+
+export type JudgingSessionConstructorValues = ConstructorValues<JudgingSession>;
```

Comment:
See my comments on the constructor
```suggestion
export type JudgingSessionConstructorValues = ConstructorValues<JudgingSession, 'inviteCode'>;
```